frontend/vre/utils/jsonld.model.js

```diff
@@ -43,6 +43,41 @@ export function getStringLiteral(literalObject) {
     }, null);
 }
 
+/**
+ * Check whether the given literal is of the given data type.
+ * @param literalObject - the object to be checked
+ * @param expectedType - the expected XSD type (part after xsd:)
+ * @returns {boolean}
+ */
+function checkDataType(literalObject, expectedType) {
```

Comment:
Maybe call it `hasDataType` to express (even) more clearly that this is the type of function that returns a boolean. `checkX` functions sometimes have more complicated semantics, for example returning nothing vs. throwing an exception.